# HG changeset patch
# User Rik <rik@octave.org>
# Date 1490024888 25200
#      Mon Mar 20 08:48:08 2017 -0700
# Branch stable
# Node ID 7f42006b94652aff3e2ff1e882b5830cda1281a6
# Parent  c0cc56ab176d31d3095f8cf48032290431bb112e
errorbar.m: move code for axes modifications AFTER plot has been made.
Change cset c0cc56ab176d to conform to general pattern in plot/draw directory.

* errorbar.m: Move code to turn 'box' on after errorbar has been plotted.

diff --git a/scripts/plot/draw/errorbar.m b/scripts/plot/draw/errorbar.m
--- a/scripts/plot/draw/errorbar.m
+++ b/scripts/plot/draw/errorbar.m
@@ -173,21 +173,21 @@ function h = errorbar (varargin)
 
   oldfig = [];
   if (! isempty (hax))
     oldfig = get (0, "currentfigure");
   endif
   unwind_protect
     hax = newplot (hax);
 
+    htmp = __errplot__ ("errorbar", hax, varargin{:});
+
     if (! ishold ())
       set (hax, "box", "on");
     endif
-    
-    htmp = __errplot__ ("errorbar", hax, varargin{:});
   unwind_protect_cleanup
     if (! isempty (oldfig))
       set (0, "currentfigure", oldfig);
     endif
   end_unwind_protect
 
   if (nargout > 0)
     h = htmp;
